src/hardware/link-mso19/protocol.c

```diff
@@ -66,12 +66,20 @@ SR_PRIV int mso_send_control_message(struct sr_serial_dev_inst *serial,
 	return ret;
 }
 
+static uint16_t mso_bank_select(const struct dev_context *devc, int bank)
+{
+	if (bank > 2) {
+		sr_err("Unsupported bank %d", bank);
+	}
+	return mso_trans(REG_CTL2, devc->ctlbase2 | BITS_CTL2_BANK(bank));
+}
+
 SR_PRIV int mso_configure_trigger(const struct sr_dev_inst *sdi)
 {
 	struct dev_context *devc = sdi->priv;
 	uint16_t threshold_value = mso_calc_raw_from_mv(devc);
 
-	threshold_value = 0x153C;
+	threshold_value = 0x200;
```

Comment:
magic constant

Reply:
I added a comment for it, which is removed in a later commit anyway so doesn't matter anyway.